packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -189,6 +189,8 @@ export const formatAmountWithSuffix = ({
         const formatted = formatNumber(value, {
           intlOptions: {
             notation: 'compact',
+            minimumFractionDigits: 1,
```

Comment:
do we have a prop to control max fraction digits too?

Reply:
We have maxFractionDigits, but do we need it ? 
What was the number that you were trying here ? @snitin315

Reply:
Let's set the max digits as `2`. You can [try with `2566.780`.](https://61c19ee8d3d282003ac1d81c-cpfmlxnxou.chromatic.com/?path=/story/components-amount--amount&args=value:2566.78;type:display;suffix:humanize)